robusta_krr/core/runner.py

```
@@ -348,9 +354,80 @@ async def run(self) -> int:
         except Exception:
             logger.exception("An unexpected error occurred")
+            publish_error(traceback.format_exc())
             return 1  # Exit with error
         else:
             return 0  # Exit with success
+
+    def _send_result(self, url: str, start_time: datetime, scan_id: str, result: Result):
+        result_dict = json.loads(result.json(indent=2))
+        _send_scan_payload(url, scan_id, start_time, result_dict, is_error=False)
+
+def publish_error(url: str, scan_id: str, start_time: str, error: str):
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)